src/mantarray_desktop_app/main.py

```diff
@@ -243,6 +246,7 @@ def main(command_line_args: List[str], object_access_for_testing: Optional[Dict[
 
         shared_values_dict = SharedValues()
 
+        shared_values_dict["fe_be_connection_success"] = False
```

Comment:
Should set this value right before `_set_up_socketio_handlers` is called. Also the name of this key should be more general, like `"websocket_connection_made"`